vpr/src/place/timing_place_lookup.cpp

```diff
@@ -77,7 +77,20 @@ static float route_connection_delay(
     const t_router_opts& router_opts,
     bool measure_directconnect);
 
-static void generic_compute_matrix(
+static void generic_compute_matrix_with_router(
+    RouterDelayProfiler& route_profiler,
+    vtr::Matrix<std::vector<float>>& matrix,
+    int source_x,
+    int source_y,
+    int start_x,
+    int start_y,
+    int end_x,
+    int end_y,
+    const t_router_opts& router_opts,
+    bool measure_directconnect,
+    const std::set<std::string>& allowed_types);
+
+static void generic_compute_matrix_with_expansion(
```

Comment:
I think 'with_router' and 'with_expansion' don't really clearly describe the differences between these two approaches.

I'd suggest something like: `generic_compute_matrix_iterative_astar()` and `generic_compute_matrix_dijkstra_flood()`.